### PR TITLE
Bug , UI and Error fixes :

### PR DESCRIPTION
**Pull request by RushikeshPatange**
- Typo fix in default_config and rainmaker_config toml's
	- style.css  -> Shade of disabled button decreased
	- custom.css -> Add :animation for alert div
	- index.html -> Add : div's for tooltip's to work and fix the footer at bottom
	- script.js  -> Fix : about us try now buttons click
	- index.js   -> Handle errors about erase,program and flash button.

### CODE REVIEW

**Comment by RushikeshPatange** [comment]
PTAL : https://rushikeshpatange.github.io/esp-launchpad/

File: js/index.js [hunk 1]
```diff
@@ -360,20 +388,29 @@ function removeRow(btnName) {
...
+    transport = undefined;
+    chip = "default";
+    reader = undefined
 }
 
 disconnectButton.onclick = async () => {
-    if(transport)
-        await transport.disconnect();
-
+    // if(transport)
```
Comment (adwait-esp) [inline js/index.js:397]: remove the commented code

File: js/index.js [hunk 2]
```diff
@@ -446,11 +502,18 @@ function validate_program_inputs() {
 }
 
 programButton.onclick = async () => {
+    programButton.disabled = true
```
Comment (adwait-esp) [inline js/index.js:505]: use standard practice of using semicolons

Comment (RushikeshPatange) [reply js/index.js]: I have added semicolons too as standard practice.PTAL

**Review by adwait-esp — APPROVED** [review]

**Pull request merged**